src/index.test.js

```diff
@@ -0,0 +1,16 @@
+jest.mock('./functionality.js');
+const { addNewTask, deleteTask } = require('./functionality.js');
+
+describe('Add and Remove', () => {
+  test('should add a new task.', () => {
+    const task2 = 'Tasklist 2';
+    const addtask = addNewTask(task2);
+    expect(addtask).toBe(1);
+  });
+
+  test('should remove one task from the array.', () => {
+    const index = 1;
+    const removetask = deleteTask(index);
+    expect(removetask).toBe(1);
+  });
+});
```

Comment:
- [ ] [OPTIONAL] You did well to test adding and removing tasks using `toBe` assertion, which checks for strict equality.  However, depending on the implementation of the addNewTask and deleteTask functions, it may be more appropriate to use other assertions such as `toContain` or `not.toContain` to check if a task was added or removed from the list.